plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/AbstractTestIcebergSmoke.java

```diff
@@ -1027,6 +1028,55 @@ public void testTruncateTextTransform()
         dropTable("test_truncate_text_transform");
     }
 
+    @Test(dataProvider = "truncateNumberTypesProvider")
+    // This particular method may or may not be @Flaky. It is annotated since the problem is generic.
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/5201", match = "Failed to read footer of file: HdfsInputFile")
+    public void testTruncateNumberTransform(String dataType)
```

Comment:
Number -> integer

number is too vague for me.
integer isn't perfect (it suggests `integer` more than the whole class of integer types), but still preferred AFAIC